internal/api/verify.go

```diff
@@ -335,6 +335,18 @@ func (a *API) signupVerify(r *http.Request, ctx context.Context, conn *storage.C
 		if terr = user.Confirm(tx); terr != nil {
 			return internalServerError("Error confirming user").WithInternalError(terr)
 		}
+
+		if identity, terr := models.FindIdentityByIdAndProvider(tx, user.ID.String(), "email"); terr != nil {
+			if !models.IsNotFoundError(terr) {
```

Comment:
why skip IsNotFoundError? in Signup flow,  user and identity will be both inserted, identity record should be found here.